core-web/libs/sdk/experiments/project.json

```diff
@@ -35,8 +35,14 @@
         },
         "publish": {
             "command": "node tools/scripts/publish.mjs sdk-experiments {args.ver} {args.tag}",
+            "outputPath": "dist/libs/sdk/experiments",
             "dependsOn": ["build", "test"]
         },
+        "nx-release-publish": {
+            "options": {
+                "packageRoot": "dist/libs/sdk/experiments"
+            }
+        },
```

Comment:
What this do?

Reply:
this fixes the path of the files used to `publish`